### DISCUSSION TITLE
How to set a custom active-color for q-tabs

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#8787] [General - Components / Directives / etc] [answered]
I am not able to find a way to set a custom color for the active tab or for the indicator (bottom border) without changing the Quasar variables defaults and without overriding quasar's CSS.. 

So what I wished I could do, is to pass my color hex to the props directly like this:

`  <q-tabs
        no-caps
        indicator-color="#0077FF"
        active-color="#0077FF"
      >....</q-tabs>`

But this doesn't work, so how do you handle such a case?

**Comment by metalsadman** [comment, marked as answer]
use a class https://quasar.dev/style/color-palette#adding-your-own-colors.
https://codepen.io/metalsadman/pen/RwKZpWQ

**Reply by saifobeidat** [reply]
Thanks, it worked!
